package/MDAnalysis/core/universe.py

```diff
@@ -298,6 +298,12 @@ def __init__(self, *args, **kwargs):
         # Universes are anchors by default
         self.is_anchor = kwargs.get('is_anchor', True)
 
+    def copy(self):
+        """Return an independent copy of this Universe"""
+        new = self.__class__(self._topology.copy())
```

Comment:
eg here we need to see what other state a Universe contains in addition to the top + traj.

Things off the top of my head are things like anchoring? as well as kwargs it was initiated with